src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -749,11 +748,28 @@ private bool WarnAboutDuplicatePackageConflicts(PackageVersion package,
 
         internal async void ExecutePackageDownload(string name, PackageVersion package, string installPath)
         {
-            string msg = String.IsNullOrEmpty(installPath) ?
+            string msg;
+            MessageBoxResult result;
+
+            var compatible = PackageManagerSearchElement.CalculateCompatibility(package.compatibility_matrix); 
+            if (compatible == null || compatible == false)
```

Comment:
For nullable bool, looks like we can just use `!= true`, referring https://stackoverflow.com/questions/2673918/best-way-to-check-for-nullable-bool-in-a-condition-expression-if#:~:text=If%20you%20want%20to%20treat%20a%20null%20as%20false,%20then

Reply:
Oh, of course! Elegant.